search-config.json

```diff
@@ -164,6 +164,18 @@
       },
       "selectors_key": "hz"
     },
+    {
+      "url": "https://docs.hazelcast.com/hazelcast-commandline-client/(?P<version>.*?)/",
+      "tags": [
+        "hazelcast-commandline-client-5.2-beta-2"
+      ],
+      "variables": {
+        "version": [
+          "5.2-beta-2"
+        ]
+      },
+      "selectors_key": "command-line-client"
```

Comment:
You need to create a selector object called `command-line-client`. This is an example: https://github.com/hazelcast/hazelcast-docs/blob/30cda5f127837f075c83be56f916507694baf50a/search-config.json#L319

Reply:
Think that is it?